keys-and-keywords/ubuntu-keywords.robot

```diff
@@ -40,3 +40,22 @@ Install Package
     Write Into Terminal    apt-get install --assume-yes ${package}
     Read From Terminal Until Prompt
     Set DUT Response Timeout    30s
+
+Stop Logging To Terminal
```

Comment:
Is it used anywhere? Since we add it in this PR?

Reply:
It is not used anywhere, but the task was to add this keyword. It could jut as well be removed and `Set Logging Level` could be used instead. That is just a matter of the level of abstraction we want to have here.

Reply:
Adding unused code is a bad practice. Nothing links this code to anything else in the project. If forgotten it may cause problems in future when someone will want to use a keyword with the same name.

"Best part is no part at all"

As we are not writing a library with some declared specification...
In my opinion it should be removed.
Unless it canbe used in the code somewhere - then it should be used in this PR.

Reply:
That is true. 
The issue did not include details on where this KW could be used, but because of this PR https://github.com/Dasharo/preseeds/pull/19 the keyword should not even be needed as long as the Ubuntu installation was performed using our auto-installer in the newest version.
There is also the question if `Get/Set Logging Level` keywords should be kept. I don't know at this moment where they could be needed, if at all.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/compare/0b887780e8f42a1edd83a483217840694e9f4cce..cf93c30f34bed18e15389c9c5adc3043a8589bb8#diff-b7afbf080f275d1995a5212ce98c26d58fb1004705c786cc31496e1043ac04b6L44-L48